backend/src/prompts/templates/validator.j2

```diff
@@ -24,7 +26,8 @@ Answer: Microsoft's ESG (Environmental, Social, and Governance) scores area as f
 Response: False
 Reasoning: The answer is for Microsoft not Apple.
 
-You must always return a single boolean value as the response.
-Do not return any additional information, just the boolean value.
-
-Spending is negative
+You must return a json result:
+{

```

Comment:
Does this change from single boolean return to json not need a corresponding change in the `supervisor.py` `is_valid_answer` to check the new `is_valid` property?